experiments/zrq/ansible/16-config-yarn-masters.yml

```diff
@@ -0,0 +1,190 @@
+
+            <!--+
+                | [host]
+                | Single hostname that can be set in place of setting all yarn.resourcemanager*address resources.
+                | Results in default ports for ResourceManager components. 
+                +-->
+            <property>
+                <name>yarn.resourcemanager.hostname</name>
+                <value>{{hdhost}}</value>
+            </property>
+
+            <!--+
+                | [host:port]
+                | If set, overrides the hostname set in yarn.resourcemanager.hostname. 
+            <property>
+                <name>yarn.resourcemanager.address</name>
+                <value>{{hdhost}}:8032</value>
+            </property>
+                +-->
+
+            <!--+
+                | [host:port]
+                | If set, overrides the hostname set in yarn.resourcemanager.hostname. 
+            <property>
+                <name>yarn.resourcemanager.admin.address</name>
+                <value>{{hdhost}}:8033</value>
+            </property>
+                +-->
+
+            <!--+
+                | [host:port]
+                | If set, overrides the hostname set in yarn.resourcemanager.hostname. 
+            <property>
+                <name>yarn.resourcemanager.resource-tracker.address</name>
+                <value>{{hdhost}}:8031</value>
+            </property>
+                +-->
+
+            <!--+
+                | [host:port]
+                | If set, overrides the hostname set in yarn.resourcemanager.hostname. 
+            <property>
+                <name>yarn.resourcemanager.scheduler.address</name>
+                <value>{{hdhost}}:8030</value>
+            </property>
+                +-->
+
+            <!--+
+                | [host:port]
+                | If set, overrides the hostname set in yarn.resourcemanager.hostname. 
+            <property>
+                <name>yarn.resourcemanager.webapp.address</name>
+                <value>{{hdhost}}:8088</value>
+            </property>
+                +-->
+
+            <!--+
+                | The class to use as the resource scheduler.
+                | CapacityScheduler (recommended), FairScheduler (also recommended), or FifoScheduler.
+                | Use a fully qualified class name, e.g.
+                | org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler
+                | org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler	
+            <property>
+                <name>yarn.resourcemanager.scheduler.class</name>
+                <value>org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler</value>
+            </property>
+                +-->
+
+            <!--+
+                | Maximum limit of memory to allocate to each container request at the Resource Manager.
+            <property>
+                <name>yarn.scheduler.maximum-allocation-mb</name>
+                <value>15000</value>
```

Comment:
Perhaps a future issue, to make the max/min allocation values configurable

Reply:
I went with the defaults to start with (most of these settings are commented out in the initial deploy).
We need to have tests in place to demonstrate why we need to change these values, and be able to re-run the tests to check what happens if we change the settings.

Reply:
Added new issues to document the settings we need for Hadoop(#95) and Spark(#96).